setup.py

```diff
@@ -49,6 +49,7 @@
     files_spec = [
         ('etc/bash_completion.d', ['youtube-dl.bash-completion']),
         ('etc/fish/completions', ['youtube-dl.fish']),
+        ('etc/zsh/completions', ['youtube-dl.zsh']),
```

Comment:
zsh completions are not stored at this location. `youtube-dl.zsh` does not exist at this point.